api/public/cerbos/request/v1/request.proto

```diff
@@ -541,6 +545,10 @@ message GetPolicyRequest {
       example: "\"principal.sarah.vdefault\""
     }
   ];
+  bool include_disabled = 2 [
+    (google.api.field_behavior) = OPTIONAL,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Include disabled policies"}
+  ];
```

Comment:
Do we need this flag for `GetPolicy`? If the user gives us a valid ID that exists in the database we can simply return it, can't we?